spec/requests/posts_controller_spec.rb

```diff
@@ -1489,11 +1494,6 @@
           it "should add custom fields to topic that is permitted for a non-staff user via the deprecated `meta_data` param" do
             sign_in(user)
 
-            Discourse.expects(:deprecate).with(
-              "the :meta_data param is deprecated, use the :topic_custom_fields param instead",
-              anything,
-            )
-
```

Comment:
This conflicts with the additional deprecation warning. Couldn't make a double expectation work, so I made an executive decision to remove this instead, since it's not a critical assertion.